src/java/org/apache/cassandra/service/StartupChecks.java

```diff
@@ -136,7 +136,7 @@ public enum StartupCheckType
                                                                       checkJMXProperties,
                                                                       inspectJvmOptions,
                                                                       checkNativeLibraryInitialization,
-                                                                      initSigarLibrary,
+                                                                      checkProcessEnvironment,
```

Comment:
there is `initSigarLibrary` string in the comment of this class. You need to replace it with this check.